landscape/explanation/glossary/repositories.md

```diff
@@ -0,0 +1,5 @@
+# Repositories
+
+**Repositories** are the foundation for software distribution in Linux environments. They hold all the necessary files and resources needed for projects. Linux distributions like Ubuntu maintain repositories containing thousands of pre-built packages you can install on your managed computers.
```

Comment:
```suggestion
**Repositories** are the foundation for software distribution in Linux environments. They hold all the necessary files and resources needed for projects. Linux distributions like Ubuntu maintain package repositories containing thousands of pre-built packages you can install on your managed computers.
```

Reply:
I'm suggesting this change to make it more clear that the repos Landscape deals with are _package_ repositories (i.e., not source-code repositories).

Reply:
Yeah, that makes sense!